src/cephadm/cephadm.py

```diff
@@ -1035,17 +1035,32 @@ def get_container_binds():
         return binds
 
     @staticmethod
-    def get_version(ctx, container_id):
-        # type: (CephadmContext, str) -> Optional[str]
-        version = None
-        out, err, code = call(ctx,
-                              [ctx.container_engine.path, 'exec', container_id,
-                               '/usr/bin/python3', '-c',
-                               "import pkg_resources; print(pkg_resources.require('ceph_iscsi')[0].version)"],
-                              verbosity=CallVerbosity.QUIET)
+      def python(s: str) -> Tuple[str, str, int]:
```

Comment:
no need to change the function name/signature

Reply:
this python() function should be nested inside the get_version() function, similar to main

Reply:
Here, I'm out of my depth and need to be taught to read.